usaspending_api/etl/es_award_template.json

```diff
@@ -27,18 +22,7 @@
       }
     }
   },
-  "sort": {
-    "fields": [
-      "total_obligation",
-      "action_date"
-    ],
-    "order": [
-      "desc",
-      "desc"
-    ]
-  },
```

Comment:
What happened to this one? 

I didn't actually realize we may have had this, and thought it could be a good optimization -- to sort on ingest/index in order to help the default sorting at query time.

Reply:
So Elasticsearch 7.1 does not support index sorting when there are nested documents in the index - https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-index-sorting.html. We use nested "documents" for the storing the TAS codes, so we can't have it sorted. I honestly cannot tell if this is new with 7.1, since 6.2 doesn't seem to have this issue, but looking back at the docs for 6.2 it looks like it wasn't supported back then either.